src/esp/bindings/SensorBindings.cpp

```diff
@@ -156,8 +157,14 @@ void initSensorBindings(py::module& m) {
   py::class_<SensorSuite, SensorSuite::ptr>(m, "SensorSuite")
       .def(py::init(&SensorSuite::create<>))
       .def("add", &SensorSuite::add)
+      .def("merge", &SensorSuite::merge)
       .def("get", &SensorSuite::get, R"(get the sensor by id)");
 
+  // ==== SensorFactory ====
+  py::class_<SensorFactory>(m, "SensorFactory")
```

Comment:
This should be a factory method on the Sensorsuite, there is no reason for this particular class to be bound in Python like so. You should add python tests to see the divergent behavior.